### PR TITLE
Log malformed url error

### PR DESCRIPTION
**Pull request by sergii4**
Signed-off-by: Sergii Getman <getman.sergei@gmail.com>

### CODE REVIEW

**Comment by sergii4** [comment]
[issue](https://github.com/go-telegram-bot-api/telegram-bot-api/issues/404)

File: helpers.go [hunk 1]
```diff
@@ -417,7 +417,10 @@ func NewUpdate(offset int) UpdateConfig {
 //
 // link is the url parsable link you wish to get the updates.
 func NewWebhook(link string) WebhookConfig {
-	u, _ := url.Parse(link)
+	u, err := url.Parse(link)
+	if err != nil {
+		log.Println("malformed URL: %s, error: %s", link, err)
```
Comment (ndrewnee) [inline helpers.go:422]: Should be `log.Printf()` not `log.Println()`

**Review by ndrewnee — CHANGES_REQUESTED** [review]

File: helpers.go (hunk 1 above)
Comment (sergevonage) [reply helpers.go]: agree. updated

Comment (sergii4) [reply helpers.go]: updated

**Comment by Syfaro** [comment]
`NewWebhook` has been updated to return an error.

**Pull request closed without merging**